### ISSUE TITLE
你好 单例模式dispatch_once_t直接崩溃bug

### ISSUE THREAD

**Issue by MoShenGuo** [SDWebImage/SDWebImage#2881]
<img width="720" alt="图片" src="https://user-images.githubusercontent.com/21171688/68000765-4a1ca600-fc9c-11e9-9447-7cd1ca81a7b5.png">

你好 崩溃bug直接崩溃在单例模式里,怎么处理呢?谢谢

**Comment by dreampiggy** [comment]
Please fill the issue template, provide your SDWebImage, iOS Version, Stacktrace all useful information. Or there are no good context for developer to investigate issues.

### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.
